### DISCUSSION TITLE
Καταγραφή κανόνων συνεργασίας στους ομαδικούς οργανισμούς

### DISCUSSION THREAD

**Discussion by p20sami** [courses-ionio/sw#1234] [Q&A] [answered]
Στο 2ο παραδοτεο που ζητείται pull request προς τον οργανισμό , τι ακριβώς πρέπει να στείλουμε στον οργανισμό? Διότι στις οδηγίες δεν υπάρχει κάτι σαφές για το συγκεκριμένο παραδοτεο, πέραν των οδηγιών των συμμετοχικών και της κατασκευής.

**Comment by Angeloth1** [comment, marked as answer]
Αν δεν κάνω λάθος χρειάζεται να συζητήσεις με την ομάδα σου σχετικά με το πώς θέλετε να λειτουργήσετε στο συνεργατικό κομμάτι του μαθήματος. Για παράδειγμα μπορείτε να θέσετε κανόνες για τα pull req.

Αυτά πρέπει να αναγράφουν στον οργανισμό σε κάποιο discussion. ( Βέβαια υπάρχει πιθανότητα να μην έχω καταλάβει κάτι ορθά οπότε καλό θα ήταν να περιμένεις και την απάντηση του καθηγητή ).

**Reply by epidrome** [reply]
σωστά το ζητούμενο είναι να αρχίσετε την διαδικασία ορισμού και δημόσιας καταγραφής κανόνων συνεργασίας στους οργανισμούς σας
